bootcamp/tutorials/milvus_usages/funnel_search_with_matryoshka.ipynb

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
<p>We probably want to publish this on milvus.io docs as a standalone notebook as well. Thus I think it's worth adding a bit of intro. Something like intro  what's "funnel search" and why it's interesting, what's Matryoshka, why it can be used for that etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>